pages/home/_meta.tsx

```diff
@@ -72,6 +72,9 @@ export default {
   "serve-tools": {
     title: "Serve tools",
   },
+  "auth-providers": {
+    title: "Customizing Auth",
+  },
```

Comment:
It doesn't feel like this fits in Building Tools. Maybe under Using Arcade? It's about using our platform, not building tools specifically.

Reply:
@nbarbettini let me know what you think of @evantahler 's comment below. I'm pretty sure they don't belong in Self-Hosting. Maybe I should add that as a reason why you'd need your own apps though. 

I could see up in Using Arcade, but that feels too high up in the Nav for people to worry about that concept. I thought about in Core Concepts, but didn't like an ever-growing list being there. Integrations would be nice, but it's not geared for more than toolkits for now.

Trying to find the goldilocks zone.